lib/TelephonyCredential.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Telnyx;
+
+/**
+ * Class TelephonyCredential
+ *
+ * @package Telnyx
+ */
+class TelephonyCredential extends ApiResource
+{
+    const OBJECT_NAME = "telephony_credential";
```

Comment:
@weroh in this case, the endpoint is `telephony_credentials` however the `record_type` is `credential` 

```json
{
  "data": {
    "created_at": "2020-06-18T21:32:38",
    "expired": "false",
    "expires_at": "2042-06-18T21:32:38",
    "id": "c215ade3-0d39-418e-94be-c5f780760199",
    "name": "2020-06-18 21:32:38.917732Z",
    "record_type": "credential",
    "resource_id": "connection:804252963366242252",
    "sip_password": "a92dbcfb60184a8cb330b0acb2f7617b",
    "sip_username": "gencrednCvHU5IYpSBPPsXI2iQsDX",
    "updated_at": "2020-06-18T21:32:38.000Z",
    "user_id": "user-id"
  }
}
```

If possible, we'd like to keep the object name as `TelephonyCredential` to match the endpoint.

The function [convertToTelnyxObject](https://github.com/team-telnyx/telnyx-php/blob/master/lib/Util/Util.php#L43) is unable to match the `record_type` to this object as-is.

This is what is causing the credential tests to fail.

Reply:
The new version expects `record_type` to be credential while using `telephony_credentials` as the url